src/main/java/tds/exam/results/mappers/ExamineeMapper.java

```diff
@@ -24,12 +24,16 @@
     private static final String DOB_ATTRIBUTE_ID = "DOB";
     private static final Logger log = LoggerFactory.getLogger(ExamineeMapper.class);
 
-    private static final Map<String, String> attributeNames = ImmutableMap.of(
-        DOB_ATTRIBUTE_ID, "Birthday",
-        "Gender", "Sex",
-        "LastName", "LastOrSurname",
-        "SSID", "StudentIdentifier"
-    );
+    private static final Map<String, String> trtAttributeRelationshipTranslations = ImmutableMap.<String, String>builder()
+        .put(DOB_ATTRIBUTE_ID, "Birthday")
+        .put("Gender", "Sex")
+        .put("LastName", "LastOrSurname")
+        .put("SSID", "StudentIdentifier")
+        .put("DistrictID", "ResponsibleDistrictIdentifier")
```

Comment:
If you look at http://www.smarterapp.org/documents/TestResults-DataModel.pdf, page 8, it shows this:
```
TRT:ExamineeRelationship-DistrictId
DWSA:ResponsibleDistrictIdentifier
AIR:DistrictID 
```
Presumably this mapping is going from the legacy key, "DistrictID" to a newer key. But the TRT spec and what we currently parse for is "DistrictId", not "ReponsibleDistrictIdentifier". So, from a purely DW selfish perspective i'm wondering what/who the change is for.